src/docs/release/breaking-changes/index.md

```diff
@@ -25,6 +25,7 @@ release, and listed in alphabetical order:
 * [Rebuild optimization for OverlayEntries and Routes][]
 * [Scrollable AlertDialog][]
 * [TestTextInput state reset][]
+* [TestWidgetsFlutterBinding.clock][]
```

Comment:
@Sfshaza When I updated my branch this came out in the wash. It looks like it may have gotten lost.

Reply:
Yes, this is odd. This line is definitely there, as of now.